.gitattributes

```diff
@@ -0,0 +1 @@
+debian export-ignore
```

Comment:
I see the documentation for this but it's not clear to me what impact that this has?

Reply:
I used the following command to collect the `orig` tarball for testing the packaging:
```
$ git archive --format=tar.gz -o ../regolith-i3xrocks-config_$(dpkg-parsechangelog --show-field Version | cut -d'-' -f1).orig.tar.gz HEAD
```
it makes sure you only add files to the tarball that have been checked into git and discards any items you're "ignoring" via the attributes :slightly_smiling_face: 

I can remove it if you want to, it doesn't hurt though because it's only relevant to `git archive`.

Reply:
Ah,  I like it!   I've been brutishly doing stuff like `tar cfzv ../<pkg filename> --exclude debian --exclude .git\* --exclude .idea --exclude .vscode .`

Thanks!